app/components/teachers/details/current_induction_period_component.html.erb

```diff
@@ -13,7 +13,3 @@
     end
   end
 %>
-
-<p class='govuk-!-text-align-right govuk-!-margin-bottom-0'>
-  <%= govuk_link_to('Add an induction period', new_admin_teacher_induction_period_path(teacher_id: teacher.id)) %>
```

Comment:
Just noticed that you originally put the link in here. 

My mind was wondering back and forth thinking about where to put this, at first I thought the show page would be best but then I realised that the `InductionSummaryComponent` component is rendered from within another component 😅 

what was your reason for deciding to put it in the `InductionSummaryComponent` in the end?

Reply:
Well I have a sneaking suspicion it'll move again. @emily-prudence-dfe spotted it wasn't getting rendered for some ECTs who no longer had an ongoing induction so it needed to be in a different template. It might actually warrant its own tiny and very simple component so we can compose and position it wherever we want; but a counter argument about making it a big button at the top was thrown in the mix. I decided this is the lightest touch and gives us time to consider it.